### ISSUE TITLE
[REQUEST]: Determine wither Hardware or Software encoder is used?

### ISSUE THREAD

**Issue by ZeoWorks** [Unity-Technologies/UnityRenderStreaming#713]
### Is your feature request related to a problem?

Hi, how are we able to tell if hardware encoding is being used as opposed to software?
Can we toggle between them?

Thanks.

**Comment by karasusan** [comment]
We will make APIs for the video/audio codec configuration for the next version.
And alternative way is using WebRTC Stats API.
https://docs.unity3d.com/Packages/com.unity.webrtc@2.4/manual/statistics.html

**Comment by ZeoWorks** [comment]
So in its current state; does it automatically use hardware and if there is no support it switches to software?

**Comment by karasusan** [comment]
@ZeoWorks
No, in the latest version 3.1.0-exp.3 we can enable/disable to use hardware.
We are preparing the feature the fallback of hardware codec.

**Comment by ZeoWorks** [comment]
When will this be ready?

**Comment by karasusan** [comment]
I mean "fallback" is that users can configure the priofity of video codecs. 
If the platform doesn't implement the hardware codec, we can choose the second option automatically.

In our plan, we can support this feature in October or later.

**Comment by karasusan** [comment]
same issue
https://github.com/Unity-Technologies/com.unity.webrtc/issues/306
